content/en/agent/faq/install-the-agent-with-aws-ssm.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Install the Agent with AWS SSM
+kind: faq
+further_reading:
+- link: "agent/"
+  tag: "Documentation"
+  text: "Learn more about the Datadog Agent"
+---
+
+Follow the steps below to install the Datadog Agent on an EC2 instance with AWS Systems Manager.
+
+### EC2 instance
+* Create an [EC2 IAM Role][1] with the `AmazonEC2RoleforSSM` prebuilt policy or update your current [AWS Datadog policy][2].
+* Launch an [EC2 instance][3] with this role attached.
+
+### Parameter Store
+In the [Parameter Store][4], create a parameter with:
+
+* Name: `dd-api-key-for-ssm`
```

Comment:
Can be more generic/standard name, I chose that for identification while I was testing.

Reply:
Actually yours had your name in it. I changed it to this here and in the JSON. This way, the person following the instructions doesn't have to update the name in the JSON.